test/bigtest/network/factories/eresource.js

```diff
@@ -0,0 +1,21 @@
+import { Factory, faker } from '@bigtest/mirage';
+
+export default Factory.extend({
+  id: () => faker.random.uuid(),
+  class: () => faker.random.arrayElement(['org.olf.kb.Pkg']),
+  name: () => faker.random.words(),
+  customCoverage: () => false,
+
+
+  afterCreate(eresource, server) {
```

Comment:
This (and the `class` above, I guess), seems to suggest that all eresources are packages. I know currently they are, but we know they won't be for all tests so we're doing ourselves a disservice by not thinking about that future scenario.